charts/aro-pull-secret/Chart.yaml

```diff
@@ -5,4 +5,4 @@ maintainers:
   - name: paulczar
 home: https://github.com/rh-mobb/helm-charts
 type: application
-version: 0.1.1
+version: 0.1.2
```

Comment:
this shouldn't be needed except for the [accidental] deletion of the values file.